om-mig.html

```diff
@@ -4,20 +4,34 @@
 
     <meta charset="UTF-8" > 
     <title>Om mig</title>
-    <!-- Idag ändrade jag storkeken på bilen och ändrade på vissa rader kod  -->
+    <!-- Filip har gjort denna hemsida, 2024   -->
     <link rel="stylesheet" href="styles.css">
    
 </head>
 <body>
     <h1> Om mig </h1>
 
-    <p>Hej!</p> 
-    <p>Jag heter filip </p>
+    <p><i>Hej!</i></p>
+    <p>Jag heter filip och jag går på nti gymnasiet </p>
 
 
     <img src="TobiasCowboy.jpg" alt="TobiasCowboy" width="500" height="auto">
-    <p>Länk till nti gymnasiets hemsida </p> ↓ </p>
+    
+
+    <h2>Mina 5 favorit dryckor</h2>
+    <ol>
+     <li>Coca cola</li>
+     <li>Fanta</li>
+     <li>Vatten</li>
+     <li>Apelsinsaft</li>
+    <li>Äppelsaft</li>
+
+    </ol>  
+
+    <p><b>Länk till nti gymnasiets hemsida </b></p> ↓ </p>
     <a href="http://www.ntigymnasiet.se/sodertalje/">ntigymnasiet.se</a>
```

Comment:
Det är bättre att lägga själva länken i texten runt ordet NTI gymnasiet istället för att lägga in en pil och ytterligare en text med länk.